.github/workflows/build-android-apk.yaml

```diff
@@ -75,6 +75,28 @@ jobs:
           path: ./client/packages/android/app/build/outputs/apk/arm64/release/*.apk
           retention-days: 3
 
+      - name: Upload Universal APK to Backblaze B2
+        uses: yamatt/backblaze-b2-upload-action@v7
+        with:
+          key_id: ${{ secrets.B2_KEY_ID }}
+          application_key: ${{ secrets.B2_APPLICATION_KEY }}
+          bucket_name: ${{ secrets.B2_BUCKET_NAME }}
+          file_path: ./client/packages/android/app/build/outputs/apk/universal/release/*.apk
+          remote_path: "android/"
+
+      - name: Upload Arm64 APK to Backblaze B2
+        uses: yamatt/backblaze-b2-upload-action@v7
+        with:
+          key_id: ${{ secrets.B2_KEY_ID }}
+          application_key: ${{ secrets.B2_APPLICATION_KEY }}
+          bucket_name: ${{ secrets.B2_BUCKET_NAME }}
+          file_path: ./client/packages/android/app/build/outputs/apk/arm64/release/*.apk
+          remote_path: "android/"
+
+      - name: Install Python dependencies
+        if: always()
+        run: pip3 install requests
```

Comment:
Side quest: Our runner doesn't have python installed. Can either download through this action or just straight in Mac Mini. Fix here is to just raise the issue, open to installing in different ways

https://github.com/msupply-foundation/open-msupply/actions/runs/16039932962/job/45259362714#step:12:66

Reply:
Chatted with James. Best to install manually on runner. Will update code shortly